public/app/plugins/datasource/loki/queryHints.ts

```diff
@@ -97,5 +104,38 @@ export function getQueryHints(query: string, series: DataFrame[]): QueryHint[] {
     }
   }
 
+  const hasLabelFilter = isQueryWithLabelFilter(query);
+  const hasLogfmt = isQueryWithLogFmt(query);
```

Comment:
I am not sure if I understand why we do `isQueryWithLogFmt` check here. We already do `  if (queryWithParser) {` on line 59, so we can just add `isQueryWithLabelFilter(query)`  check into that block. Cause we want to suggest label filter after any parser, not just logfmt.